frontend/src/components/FacilityBooking/CurrentTimeLine.tsx

```diff
@@ -40,7 +32,19 @@ export function scrollToView(ref: RefObject<HTMLHRElement> | React.RefObject<HTM
   }
 }
 
-const CurrentTimeLine = (props: Props) => {
+/**
+ * @param width (type `string`)
+ * @param top (type `string`)
+ * @param left (type `string`)
+ * @param right (type `string`)
+ * @param bottom (type `string`)
+ * @returns If the selected day is today, it returns a timeline indicating the current time, where its position
+ * is updated every minute. Otherwise, it returns nothing.
+ * @example // <CurrentTimeLine/> is used in the <BookingSection/> and <ViewScheduleBlock/> components to indicate the current time.
```

Comment:
for @example right, you can write actual TypeScript code in it
you can refer to the [docs](https://www.notion.so/rhdevs-wiki/How-To-Frontend-Documentation-390a6a02b00d4eb59fd7fa9563025885) here! (refer to **Components** and the `@example` in the screenshot)